io/zenoh-links/zenoh-link-quic/src/unicast.rs

```diff
@@ -143,6 +143,12 @@ impl LinkUnicastTrait for LinkUnicastQuic {
         *QUIC_DEFAULT_MTU
     }
 
+    #[inline(always)]
+    fn get_interfaces(&self) -> Vec<String> {
+        // Not supported for now
+        vec![]
```

Comment:
As a general comment, we start having many links implementations that rely on `IPv4/IPv6`.
I wonder wether having some shared `ip utils` or even an `IPtrait` for IP-based links would be better.